miscellaneous_scripts/dockerfile_patch_script.sh

```diff
@@ -29,6 +31,20 @@ mv $PATCHING_INFO_PATH/patch-details-current $PATCHING_INFO_PATH/patch-details
 chmod +x $PATCHING_INFO_PATH/patch-details/install_script_language.sh && \
 $PATCHING_INFO_PATH/patch-details/install_script_language.sh
 
+if [ $LATEST_RELEASED_IMAGE_URI == "763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:2.0.1-gpu-py310-cu121-ubuntu20.04-sagemaker" ]; then
+    SMP_URL=https://smppy.s3.amazonaws.com/pytorch/cu121/smprof-0.3.334-cp310-cp310-linux_x86_64.whl
+    pip install --no-cache-dir -U ${SMP_URL}
+    apt-get install --only-upgrade libxml2
```

Comment:
Does autopatch support apt-get installs? Why do we have to do this conditionally?